tools/java/org/unicode/cldr/util/data/PathDescription.txt

```diff
@@ -103,6 +103,8 @@
 
 ^//ldml/numbers/minimalPairs/ordinalMinimalPairs\[@ordinal="([^"]*)"] ; Minimal pairs for ordinals. For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
 ^//ldml/numbers/minimalPairs/pluralMinimalPairs\[@count="([^"]*)"] ; Minimal pairs for plurals (cardinals). For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
+^//ldml/numbers/minimalPairs/caseMinimalPairs\[@case="([^"]*)"] ; Minimal pairs for cases used in the language. For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
+^//ldml/numbers/minimalPairs/caseMinimalPairs\[@gender="([^"]*)"] ; Minimal pairs for genders. For more information, please see http://cldr.org/translation/getting-started/plurals#TOC-Minimal-Pairs.
```

Comment:
ldml/numbers/minimalPairs/caseMinimalPairs\[@gender

from the dtd, this should be
ldml/numbers/minimalPairs/genderMinimalPairs\[@gender

Reply:
Thanks, just checked in fix.